spec/features/user_spec.rb

```diff
@@ -12,35 +13,45 @@
         expect(page).to have_content(user.name)
       end
 
-      it 'allows create cards' do
-        create_card
+      it 'allows to create cards in deck' do
+        visit_create_card_path
+        card_fill_in
+        select deck.name, from: 'Deck'
+        click_button 'Создать'
         expect(page).to have_content('Карточка создана')
       end
 
       it 'not allows create empty cards' do
-        click_link 'Добавить карточку'
+        visit_create_card_path
         click_button 'Создать'
         expect(page).to have_content('Заполните все поля!')
       end
 
       it 'allows edit cards' do
         create_card
-        visit cards_path
         click_link 'Редактировать'
         fill_in :card_original_text, with: 'Another word'
         click_button 'Создать'
         expect(page).to have_content('Карточка успешно отредактирована!')
       end
 
-      it 'allows edit profile' do
+      it 'allows to edit profile' do
         click_link user.name
-        expect(page).to have_button('Обновить')
+        fill_in :user_name, with: 'qwerty'
+        fill_in :user_password, with: 123456
+        fill_in :user_password_confirmation, with: 123456
+        click_button 'Обновить'
+        expect(page).to have_content('Данные обновлены')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.